src/modules/sidebar/sections/SidebarSharing.vue

```diff
@@ -38,6 +41,13 @@ export default {
 
 	computed: {
 		...mapState(useTablesStore, ['activeElement', 'isView']),
+		currentUrl() {
+			const url = this.activeElement
+				? `${window.location.origin}/index.php/apps/tables/#/table/${this.activeElement.id}`
```

Comment:
We have two different URLs that could be shared from this sidebar, for tables and views. views would be a subset of a table that then have `#/view/:viewId` as their route.

You should be able to test this using the default table and the `Checkyourself!` example view

<img width="1502" alt="Screenshot 2025-06-03 at 09 26 52" src="https://github.com/user-attachments/assets/aa091bbe-6be8-4824-9416-51e5a7d8232c" />